static/js/compose.js

```diff
@@ -423,6 +429,10 @@ export function initialize() {
         } else {
             $("#compose_close").attr("data-tooltip-template-id", "compose_close_tooltip");
         }
+
+        if (compose_text_length <= page_params.max_message_length) {
+            compose_validate.check_send_permissions();
+        }
```

Comment:
Here is the updated check for making sure that `check_send_permissions` doesn't re-enable the send button when `max_message_length` has been exceeded.

Reply:
@sahil839 what do you think about this? It feels a little redundant since `compose_validate.check_overflow_text` is called right before this check, perhaps it's okay though since it keeps these checks in separate functions.